code/src/robos_core.cpp

```diff
@@ -77,20 +77,45 @@ namespace r2d2::robos {
             // robos_core_c::robos role = moving_platform;
             break;
         case EXAMPLE_ROLE:
+            // auto moving_platform_role = robos_role_c();
+            // robos_core_c::robos role = moving_platform;
             break;
         }
+
         return 0;
     };
 
     int robos_core_c::run_role() {
+        const std::vector<frame_type> test_frame;
```

Comment:
vector should be an array

Reply:
This is an exception made by the leads that in this case a vector is correct. So i will approve

Reply:
Ok